website/docs/docs/bi-directional-contract-testing/compatibility-checks.md

```diff
@@ -14,28 +13,178 @@ calls may have breaking changes.
 
 :::
 
-The [detail screen](../user-interface/bi-directional) for the BDC interaction has different tabs that display the contracts involved and the status of the verification. The interaction could be in an invalid (or failed) state due to the following three conditions:
+The [detail screen](/docs/user-interface/bi-directional) for the BDC interaction has different tabs that display the contracts involved and the status of the verification. The interaction could be in an invalid (or failed) state due to the following three conditions:
 
 1. The provider self-verification test results indicate that they have failed. This means that the provider build ran some tests against the OAS and the result was a failure.
 2. The consumer Pact verification results have failed. This means the consumer Pact tests failed, and the consumer is not compatible with the published Pact file.
 3. The consumer Pact file is not compatible with the OAS.
 
-## Contract Comparison
-
-This tab will display the details of the comparison between the Pact file and the OAS.
-
-## Contract Incompatibility Messages
-
-| Error Messsage | Description |
-| ---------- | ----------- |
-| Request accept header is incompatible | Pact request's accept header content is incompatible with provider contract |
-| Request misses authorization header | Pact request's authorization header is missing |
-| Request body is incompatible | Pact request's body content is incompatible with provider contract |
-| Request content type header is incompatible | Pact request's content type header is incompatible with provider contract |
-| Request header is incompatible | Pact request's header content is incompatible with provider contract|
-| Request contains unknown path or method | Pact contains unknown request's path or method compared with provider contract |
-| Request query is incompatible | Pact request query content is incompatible with provider contract |
-| Response body is incompatible | Pact response's body content is incompatible with provider contract |
-| Response body contains unknown information | Pact response's body contains unknown content compared with provider contract|
-| Response request is incompatible | Pact response's request content is incompatible with provider contract|
-| Reponse header contains unknown information | Pact response's header contains unknown content compared with provider contract |
```

Comment:
This near-English to English translation has been removed recently. It was: https://github.com/pactflow/pact-broker-react-ui/blob/9b855a129b5b05e193b42f8045d4bbb8143da2ef/src/utils/cross-verification-error-type.ts

Now, the raw error codes from SMV is displayed in the UI. Ie, things like `request.accept.incompatible`.

I removed that because the error map wasn't complete at the time of removal, and rather than chasing the list with changes to SMV (which we maintain now), I thought the raw error codes were informative enough.